toDo.md

```diff
@@ -0,0 +1,41 @@
+# <u>Issues and Steps to Resolve Them</u>
+
+Here are 5 issues that can be resolved in this code by someone at a React beginner level, along with the steps required to complete each task:
+
+### Add functionality for adding new tasks:
+
+1. Create a state variable in the Form component to store the list of tasks.
+2. Implement an onChange event listener on the input field to update the value of a separate state variable to track the input field's value.
+3. Implement an onSubmit event listener on the form that adds the new task to the task list state variable and clears the input field's value.
+4. Update the rendering of the task list to display the dynamically generated tasks from the state variable.
+
+### Implement functionality for editing and deleting tasks:
+
+1. Create a state variable in the Form component to store the list of tasks.
+2. Implement an onClick event listener on the "Edit" button that toggles the editable state of the corresponding task.
+3. Implement an onChange event listener on the corresponding task's input field to update the value of the task.
+4. Implement an onClick event listener on the "Delete" button that removes the corresponding task from the task list state variable.
```

Comment:
Linethrough the task to mark as completed and vanish after some sec...
What's your view?